connection.impl/src/main/java/com/ericsson/bss/cassandra/ecchronos/connection/impl/LocalNativeConnectionProvider.java

```diff
@@ -157,7 +157,7 @@ private static String resolveLocalDataCenter(Builder builder)
             throw new IllegalStateException("Unable to find local data center");
         }
 
-        private static Cluster.Builder fromBuilder(Builder builder)
+        public static Cluster.Builder fromBuilder(Builder builder)
```

Comment:
If the previously mentioned change is done this change shouldn't be necessary anymore.